config/lava/boot/generic-qemu-boot-template.jinja2

```diff
@@ -31,6 +31,12 @@
 {% endblock %}
 {% block actions %}
 
+{% if context %}
+context:
+{% for key, value in context.items() %}  {{ key }}: {{ value }}
+{% endfor %}
+{% endif %}
+
```

Comment:
But this is already done in the base template:
https://github.com/kernelci/kernelci-core/blob/main/config/lava/base/kernel-ci-base.jinja2#L36

And this template inherits from it.  Could you please compare the job definitions generated with and without this patch?

As far as I can tell, the context is already included as expected in QEMU jobs:
https://lava.collabora.dev/scheduler/job/11436140/definition#defline75

Reply:
Eh, this is true. I tried again, and read `generic-qemu-boot-template.jinja2`. It's my yaml file problem. Sorry for your time.